snippets/nuxt-script.json

```
@@ -0,0 +1,27 @@
+{
+  "Nuxt Async": {
+    "prefix": "nasyncdata",
+    "body": ["async asyncData ({ ${1:params} }) {", "\tconst { data } = await axios.get(`${2:endpoint}`)", "\treturn { ${3:key}:${4:value} }", "},"],
+    "description": "Nuxt asyncData"
+  },
+  "Nuxt Async Promise": {
+    "prefix": "nasyncdata-p",
+    "body": ["asyncData ({ ${1:params} }) {", "\treturn axios.get(`${2:endpoint}`)", "\t\t.then((res) => {", "\t\t\t\treturn { ${3:key}:${4:res.data} }", "\t\t})", "},"],
+    "description": "Nuxt asyncData - Returning a Promise"
+  },
+  "Nuxt Fetch": {
+    "prefix": "nfetch",
+    "body": ["async fetch ({ store, ${1:params} }) {", "\tlet { data } = await axios.get('${2:endpoint}')", "\tstore.commit('${3:MUTATION_TYPE}', data)", "},"],
```

```suggestion
    "body": ["async fetch ({ store, ${1:params} }) {", "\tlet { data } = await fetch('${2:endpoint}').then(res => res.json())", "\tstore.commit('${3:MUTATION_TYPE}', data)", "},"],
```

Same here to remove axios dependency :)